index.html

```diff
@@ -26,49 +26,289 @@
+          </div>
+        </div>
+      </div>
+    </section>
+    <section class="section-3">
+      <div class="container">
+        <div class="about-me-2">
+          <div class="headline">
+            <h1>
+              About <br />
+              Myelf
+            </h1>
+          </div>
+          <div class="about-me">
+            <p class="abt-me">
+              I’m a software developer! I can help you build a product , feature
+              or website Look through some of my work and experience! If you
+              like what you see and have a project you need coded, don’t
+              hestiate to contact me.
+            </p>
+          </div>
+          <a class="meet">LET'S CONNECT</a>
+          <div class="social-icons">
+            <ul class="social-icons-list">
+              <li>
+                <a href="#"><img src="assests/images/Vector.png" alt="" /></a>
+              </li>
+              <li>
+                <a href="#"
+                  ><img src="assests/images/Vector (1).png" alt=""
+                /></a>
+              </li>
+              <li>
+                <a href="#"
+                  ><img src="assests/images/Vector (2).png" alt=""
+                /></a>
+              </li>
+              <li>
+                <a href="#"
+                  ><img src="assests/images/Vector (3).png" alt=""
+                /></a>
+              </li>
+              <li>
+                <a href="#"
+                  ><img src="/assests/images/Linkedin icon.png" alt=""
+                /></a>
+              </li>
+            </ul>
+            <button class="buttn bttn-2">Get my resume</button>
+          </div>
+
+          <div class="skill-set">
+            <p>Languages</p>
+            <img src="/assests/images/Disabled.png" alt="" class="disable" />
+            <div class="langs">
+              <div class="javaS">
+                <img
+                  src="/assests/images/Ellipse 1.png"
+                  alt=""
+                  class="lang-icon"
+                />
+                <p class="lang-font">JavaScript</p>
+              </div>
+              <div class="htmL">
+                <img
+                  src="/assests/images/Ellipse 1 (1).png"
+                  alt=""
+                  class="lang-icon"
+                />
+                <p class="lang-font">HTML</p>
+              </div>
+              <div class="csS">
+                <img
+                  src="/assests/images/Ellipse 1 (2).png"
+                  alt=""
+                  class="lang-icon"
+                />
+                <p class="lang-font">CSS</p>
+              </div>
+            </div>
+            <div class="frame-skills">
+              <div class="framework">
+                <p>Frameworks</p>
+                <img src="/assests/images/Union.png" alt="" />
+              </div>
+              <img src="/assests/images/Separator Bottom.png" alt="" /> <br />
```

Comment:
Kindly, I suggest that for accessibility every time you use an "IMG" you indicate an appropriate "ALT" to describe the content of that image